stable/prisma/ci/example-values.yaml

```diff
@@ -0,0 +1,57 @@
+# ------------------------------------------------------------------------------
+# Prisma:
+# ------------------------------------------------------------------------------
+
```

Comment:
The file should only contain changes over the default values.yaml. Don't copy everything.